lib/hyrax/configuration.rb

```diff
@@ -373,6 +373,12 @@ def subject_prefix
       @subject_prefix ||= "Contact form:"
     end
 
+    attr_writer :extract_full_text
+    def extract_full_text?
+      return @extract_full_text unless @extract_full_text.nil?
+      @extract_full_text = true
+    end
```

Comment:
An alternative pattern for this method body could use a ternary expression, i.e.
```ruby
x.nil? ? x = true : x
```
The assignment in the ternary expression is valid and it returns `true`.